app/utils/analyzeTrackedUrl.js

```diff
@@ -15,7 +15,7 @@ export default function analyzeTrackedUrl(params) {
 	const actualUrl = window.location.href;
 
 	if (actualUrl.indexOf(gaPage) === -1) {
-		logEvent('GA url does not match window.location', Object.assign({
+		logDebug('GA url does not match window.location', Object.assign({
```

Comment:
Do we still need this logging? if yes let's create a ticket to remove it in the future.

Reply:
Yeah, we need to preserve it to see if our fixes around tracked GA urls are working. https://wikia-inc.atlassian.net/browse/XW-4850